pkgs/by-name/zu/zug/package.nix

```
@@ -15,6 +16,12 @@ stdenv.mkDerivation rec {
     rev = "v${version}";
     hash = "sha256-7xTMDhPIx1I1PiYNanGUsK8pdrWuemMWM7BW+NQs2BQ=";
   };
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/arximboldi/zug/pull/42.patch";
```

```suggestion
      url = "https://github.com/arximboldi/zug/commit/c8c74ada30d931e40636c13763b892f20d3ce1ae.patch";
```
Better to reference the commit and not the entire PR.